vault/ha.go

```diff
@@ -403,13 +418,22 @@ func (c *Core) waitForLeadership(newLeaderCh chan func(), manualStepDownCh, stop
 		if leaderLostCh == nil {
 			return
 		}
+
+		if atomic.LoadUint32(c.neverBecomeActive) == 1 {
+			c.heldHALock = nil
+			lock.Unlock()
+			c.logger.Info("marked never become active, giving up after interrupting perf standbys")
```

Comment:
Is this log true? Seems like we are not interrupting here

Reply:
Ah, it got moved. Will fix.

Reply:
```suggestion
			c.logger.Info("marked never become active, giving up active state")
```